daisy_workflows/image_import/import_image.sh

```diff
@@ -41,6 +41,11 @@ SOURCE_SIZE_BYTES="$(gsutil du "${SOURCE_URL}" | grep -o '^[0-9]\+')"
 SOURCE_SIZE_GB=$(awk "BEGIN {print int(((${SOURCE_SIZE_BYTES}-1)/${BYTES_1GB}) + 1)}")
 IMAGE_PATH="/daisy-scratch/$(basename "${SOURCE_URL}")"
 
+# Validate input
+
+[[ -z "$SCRATCH_DISK_SIZE_GB" ]] && echo "ImportFailed: metadata scratch_disk_size_gb is unset" && exit 1
+[[ -z "$INFLATED_DISK_SIZE_GB" ]] && echo "ImportFailed: metadata inflated_disk_size_gb is unset" && exit 1
```

Comment:
minor: unset -> not set. Unset sounds (at least to me) like we just cleared this metadata key (un-set it), while we want to say it's not set at all (missing).

Reply:
done!